media/js/firefox/private-browsing.js

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+;(function($, Modernizr, site) {
+    'use strict';
+
+    var $html = $(document.documentElement);
+    var $shield = $('#tracking-protection-animation');
+    var panels = ['tracking-protection', 'when-to-use', 'control-center', 'cta'];
+
+    $.each(panels, function(index, value) {
+        new Waypoint({
+            element: document.getElementById(value),
+            handler: function(direction) {
+                if(direction == 'down') {
+                    registerGA(value);
+                }
+            },
+            offset: 'bottom-in-view'
+        });
+    }); 
+
+    function registerGA(section) {
+        window.dataLayer.push({
+            event: 'private-browsing-page-interactions',
+            interaction: 'scroll',
+            section: section        
+        });
+    }
+
+    Mozilla.HighlightTarget.init('.button-flat-dark');
+
+    if (window.isFirefox()) {
```

Comment:
Just a side note: this will return true for Firefox for Android as well as Desktop. Perhaps this is intentional, but it's worth pointing out that the UITour functionality will only work on the desktop browser. I'm not sure what Firefox for Android users should see here? (sorry for the drive-by comment).